### ISSUE TITLE
Update addons for React v0.14.0

### ISSUE THREAD

**Issue by swashcap** [bebraw/reactabular#96]
With React v0.14.0 I see the following warning in the console:

```
Warning: require('react/addons') is deprecated. Access using require('react-addons-{addon}') instead.
```

I’m guessing this is referencing lines like [this](https://github.com/bebraw/reactabular/blob/209354b710dd28ca33196df0e9b4f919ebe6588b/src/editors/input.js#L3) and [this](https://github.com/bebraw/reactabular/blob/209354b710dd28ca33196df0e9b4f919ebe6588b/src/editors/dropdown.js#L3).

**Comment by bebraw** [comment]
Yeah, exactly.

I did a preliminary fix at `dev`. Do you want to give it a go? `npm i bebraw/reactabular#dev` might work I think.

**Comment by swashcap** [comment]
Ah, nice. Looks like _dist-modules_ wasn’t updated with the transpiled code, so the warning still exists. Running `npm run dist && npm run dist-min && npm run dist-modules` does the trick.

**Comment by bebraw** [comment]
The problem should be fixed in `0.8.3`. npm was glitchy while publishing, that's why `0.8.3`. Got hit by https://github.com/npm/npm-www/issues/537 .
